glossary/es/index.md

```diff
@@ -1,98 +1,6 @@
 ---
-redirect_from: /es/glossary.html
-section: glossary
+redirect_from: /en/glossary.html
+layout: glossary
```

Comment:
@ecedenyo This should be like the below:

    redirect_from:
      - /es/glossary.html
      - /glossary/es/terms/
      - /glossary/es/
    layout: glossary